services/api-server/src/simcore_service_api_server/api/routes/files.py

```diff
@@ -143,6 +143,8 @@ async def list_files(
     "/page",
     response_model=Page[File],
     include_in_schema=API_SERVER_DEV_FEATURES_ENABLED,
+    status_code=status.HTTP_501_NOT_IMPLEMENTED,
+    response_description="Not implemented",
```

Comment:
MINOR: The response is not implemented? :-) 
I would rather add that in the description of the entrypoint than in the description of the response. 

Nonetheless, IMO there is no need to verbose so much since 501 is clear enough. If not I would rather automatically produce the description converting to an [HttpStatus enum](https://docs.python.org/3/library/http.html#http.HTTPStatus) and using
 `http.HttpStatus(status_code).description`

Reply:
Done

Reply:
I meant automatically :-) but ok